packages/cozy-harvest-lib/src/connections/accounts.js

```diff
@@ -187,3 +187,17 @@ export const fetchAccountsFromTriggers = async (client, triggers) => {
     .filter(Boolean)
     .map(account => ({ account, trigger: accountIdToTrigger[account._id] }))
 }
+
+/**
+ * Returns the list of accounts which do not have any associated trigger
+ *
+ * @param  {Object}  client  CozyClient
+ * @param  {Array}  triggers io.cozy.triggers documents
+ */
+export const fetchAccountsWithoutTriggers = async (client, triggers) => {
+  const accountCol = client.collection('io.cozy.accounts')
+  const accountIdToTrigger = keyBy(triggers, triggersModel.getAccountId)
+  return (await accountCol.all({ limit: null })).data.filter(
+    account => !accountIdToTrigger[account.id]
+  )
```

Comment:
Can't we use a mango selector instead of fetching all the accounts? Does `fetchAccountsWithoutTriggers` will be called often? Since expensive 

Something like should work, no ? 
where({$not: { account : { $in : accountIdToTrigger } } }

Reply:
This function is called only when creating a new account. If I had a selector to the request, I think I would have to handle paging and then I think this is a good compromise since this request is not so expensive and not called so often.

Reply:
>  If I had a selector to the request,

You are already adding a selector here right ?

Reply:
Yes, I finally added a selector and used queryAll finally, which is fine